source/blog/2022-01-12-announcing-hanami-200alpha5.html.markdown

```diff
@@ -0,0 +1,206 @@
+---
+title: Announcing Hanami v2.0.0.alpha5
+date: 2022-01-12 11:00:00 UTC
+tags: announcements
+author: Tim Riley
+image: true
+excerpt: >
+  Sensible default logger, full source dirs config, router loading flexibility, route helpers in views
+---
+
+Happy new year, Hanami community! To get 2022 started, we're excited to announce the release of Hanami 2.0.0.alpha5!
+
+This release brings the last month of our work on Hanami 2.0 (with an extra week added for good measure, while we all returned from our end of year breaks). It includes:
+
+- Sensible default configuration for the application logger
+- Comprehensive `source_dirs` configuration
```

Comment:
```suggestion
- Comprehensive `config.source_dirs` configuration (for advanced users)
```

Reply:
👍🏼 

@jodosha FWIW I've changed this to just "source dirs" instead of `source_dirs`, to make it a more natural-reading title.